tests/e2e/specs/out-of-game/lobby.spec.js

```diff
@@ -5,11 +5,14 @@ import {
   opponentPassword,
 } from '../../support/helpers';
 
-function setup() {
+function setup(isRanked = false) {
   cy.wipeDatabase();
   cy.visit('/');
   cy.signupPlayer(validUsername, validPassword);
-  cy.createGamePlayer({ gameName: 'Test Game', isRanked: false }).then((gameSummary) => {
+  // This is required to show the game list (which will ensure the `addGameToList` action dispatches)
+  // Otherwise, tanked game creation does not register properly
+  cy.visit('/');
```

Comment:
Really weird edgecase bug that was preventing me from making a ranked game for testing as `addGametoList` wasn't firing since the "user" wasn't on the proper gamelist page. You have to revisit it to force the reflow, probably from us using the store directly.

Reply:
This smells. The reason that we 'need' the current game to appear in the `gameList` store module is because the `isRanked` getter is reading the ranked-ness from across store modules, right? Instead, it would be cleaner to have `isRanked` be a piece of state in _this_ module, describing whether the current game is ranked, and then we wouldn't need our game to appear in the `gameList` in order to correctly register it as ranked (and we could skip the extra `cy.visit('/');`.

Reply:
This is definitely a symptom of the way I implemented the change and not the direct cause. Once the `isRanked` property is pulled in to the game object and referenced directly I don't think we'll have this issue and it can be removed.